document_url/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+ * go to menu more and click on add an url.
+ * Insert name and url into the wizard.
```

Comment:
Remove the first blank space in these two lines or list are considered in a quote.